src/target.jl

```diff
@@ -3,6 +3,15 @@
 
 #using Distributions
 
+struct TESSTarget
+  sys::Vector{PlanetarySystemAbstract}
+  snr::Array{Float64,1} # the SNR of observations in each sector?
```

Comment:
I could easily imagine that different sectors would have different noise levels.
On a related note, will you have different noise levels depending on the transit duration?  
Also, I'm a little worried that snr might be a confusing name, if a reader thought you meant SNR of a transit, rather than SNR of a photometric measurement.